.github/workflows/ci.yaml

```diff
@@ -75,7 +75,7 @@ jobs:
 
       - name: Start delphi_web_epidata
         run: |
-          docker run --rm -d -p 10080:80 --env "MODULE_NAME=delphi.epidata.server.main" --env "SQLALCHEMY_DATABASE_URI=mysql+mysqldb://user:pass@delphi_database_epidata:3306/epidata" --env "FLASK_SECRET=abc" --env "FLASK_PREFIX=/epidata" --env "REDIS_HOST=delphi_redis" --env "REDIS_PASSWORD=1234" --env "API_KEY_REGISTER_WEBHOOK_TOKEN=abc" --env "API_KEY_ADMIN_PASSWORD=test_admin_password" --network delphi-net --name delphi_web_epidata delphi_web_epidata
+          docker run --rm -d -p 10080:80 --env "MODULE_NAME=delphi.epidata.server.main" --env "SQLALCHEMY_DATABASE_URI=mysql+mysqldb://user:pass@delphi_database_epidata:3306/epidata" --env "FLASK_SECRET=abc" --env "FLASK_PREFIX=/epidata" --env "REDIS_HOST=delphi_redis" --env "REDIS_PASSWORD=1234" --env "API_KEY_REGISTER_WEBHOOK_TOKEN=abc" --env "API_KEY_ADMIN_PASSWORD=test_admin_password" --env "TESTING_MODE=True" --network delphi-net --name delphi_web_epidata delphi_web_epidata
```

Comment:
do we wanna make this an option in the `Makefile` as well?  (we definitely want to make it default in the Makefile)

another way to achieve the same effect (and only for tests that need to reach the request limiting threshold) would be to write into redis at the beginning of those tests to simulate a large number of requests...  if thats not too difficult to do, itll be more robust.

Reply:
Actually I believe that I have added this variable to Makefile as well, but for some reason it is not there now. Let me explore this a little bit more.